### PR TITLE
increase cache length on AUR version badge, run [AUR]

### PR DESCRIPTION
**Pull request by calebcartwright**
Refs #7863

### CODE REVIEW

File: services/aur/aur.service.js [hunk 1]
```diff
@@ -108,7 +108,7 @@ class AurVotes extends BaseAurService {
 
 class AurVersion extends BaseAurService {
   static category = 'version'
-
+  static _cacheLength = 3600
```
Comment (calebcartwright) [inline services/aur/aur.service.js:111]: Open to alternative suggestions, but I'm very intentionally swinging fairly big here. Of the >18k AUR requests we've had over the last 24 hours, nearly 16k of those were for the version badges.

I realize that version badges tend to be more sensitive to recency needs, but I think we'll need to be aggressive here if we're to have a chance of staying within AURs 4k/day limit

**Review by chris48s — APPROVED** [review]
might as well give it a go..

**Comment by calebcartwright** [comment]
Is circle not working on in-repo PRs now too??

**Comment by chris48s** [comment]
It is the first time I've seen it happen on one that is not from a fork, but apparently so..

**Review by chris48s — APPROVED** [review]

**Pull request merged**